src/objectid.ts

```diff
@@ -290,34 +290,15 @@ export class ObjectId {
    *
    * @param id - ObjectId instance to validate.
    */
-  static isValid(id: number | string | ObjectId | Uint8Array | ObjectIdLike): boolean {
+  static isValid(id: string | number | ObjectId | ObjectIdLike | Buffer | Uint8Array): boolean {
```

Comment:
Was adding support for `Uint8Array`s intentional?  I notice it's added here and in the constructor

Reply:
I think so -- Since we have parity between `isValid` and the `constructor`, the constructor now needs to take in an `Uint8Array` in order for it to be called from `isValid`. Buffer is a subclass of `Uint8Array`, so nothing will be broken by allowing the more general `Uint8Array` as a param